versions.tf

```diff
@@ -27,9 +27,9 @@ terraform {
     }
   }
   provider_meta "google" {
-    module_name = "blueprints/terraform/terraform-google-cloud-armor/0.1.0"
+    module_name = "blueprints/terraform/terraform-google-cloud-armor/v0.3.0"
   }
   provider_meta "google-beta" {
-    module_name = "blueprints/terraform/terraform-google-cloud-armor/0.1.0"
+    module_name = "blueprints/terraform/terraform-google-cloud-armor/v0.3.0"
```

Comment:
Also, you don't need to increment this yourself. release-please will update this automatically when it kicks off a new release build.